editor/editor_data.cpp

```diff
@@ -435,9 +435,14 @@ void EditorData::restore_editor_global_states() {
 
 void EditorData::paste_object_params(Object *p_object) {
 
-	for (List<PropertyData>::Element *E = clipboard.front(); E; E = E->next()) {
-
-		p_object->set(E->get().name, E->get().value);
+	if (p_object) {
```

Comment:
I'd make this `ERR_FAIL_NULL(p_object);` instead, or is there a valid workflow where this method would be called with an invalid object?